example-model/src/main/java/uk/gov/gchq/palisade/example/model/client/ExampleSimpleClient.java

```diff
@@ -45,6 +45,11 @@ public void run(final String filename, final String userId, final String purpose
 
     public Stream<Employee> read(final String resourceId, final String userId, final String purpose) throws IOException {
         String absoluteFile = new File(resourceId).toURI().toString();
+        // Maintain is-a-file-or-a-directory consistency between relative and absolute paths
```

Comment:
this comment doesn't really tell me what this is doing

Reply:
updated comment